### PR TITLE
SAS - new dependency and change config so odbc works w/postgres

### PR DESCRIPTION
**Pull request by stirlingv**

### CODE REVIEW

File: deploy/roles/sas/tasks/main.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ---
 - name: Install yum dependencies
   yum:
-    name: libXext.i686
+    name: "{{ item }}"
     state: present
-  when: install_sas
+  with_items:
+    - libXext.i686
```
Comment (tanderegg) [inline deploy/roles/sas/tasks/main.yml]: @stirlingv We should take another look at why this is necessary, as it doesn't make sense to install an i686 package on an x86_64 CPU...

File: deploy/roles/sas/tasks/main.yml [hunk 2]
```diff
@@ -36,6 +38,24 @@
   tags:
     - sas
 
+- name: check sasenv script exists sas-console
+  stat:
+    path: "{{ sas_home }}/SASFoundation/9.4/bin/sasenv_local"
```
Comment (tanderegg) [inline deploy/roles/sas/tasks/main.yml]: @stirlingv Shouldn't we use `sas_version` here instead of 9.4?

File: deploy/roles/sas/tasks/main.yml [hunk 3]
```diff
@@ -36,6 +38,24 @@
...
+  register: setenv_file_console
+  tags:
+    - sas
+
+- name: Change the library path sas-console
+  lineinfile:
+    dest: "{{ sas_home }}/SASFoundation/9.4/bin/sasenv_local"
```
Comment (tanderegg) [inline deploy/roles/sas/tasks/main.yml]: `sas_version` here too?

**Review by tanderegg — CHANGES_REQUESTED** [review]
A few small issues, thanks for doing this quick!

File: deploy/roles/sas/tasks/main.yml (hunk 1 above)
Comment (stirlingv) [reply deploy/roles/sas/tasks/main.yml]: @tanderegg I think you're right. I removed dependency for .i686 library

**Comment by tanderegg** [comment]
@stirlingv Can you manually remove the libXext.i686 package from the Dev SAS VM, so we can confirm that it is not needed?  Thanks!

File: deploy/roles/sas/tasks/main.yml (hunk 2 above)
Comment (stirlingv) [reply deploy/roles/sas/tasks/main.yml]: @tanderegg yup!

File: deploy/roles/sas/tasks/main.yml (hunk 3 above)
Comment (stirlingv) [reply deploy/roles/sas/tasks/main.yml]: @tanderegg double yup!

**Comment by stirlingv** [comment]
@tanderegg removed libXext.i686 in dev. Was able to open SAS, no issues. I think we're good!

**Pull request merged**